backend-api/src/functions/asyncActiveSession/tokenService/tokenService.ts

```diff
@@ -4,7 +4,6 @@ import {
   SuccessfulHttpResponse,
```

Comment:
We can delete `getJweComponents` and `decryptJwe` functions from this file now, they are unused